docs/CHANGELOG.md

```diff
@@ -4,18 +4,26 @@ Test fixtures for use by clients are available for each release on the [Github r
 
 **Key:** ✨ = New, 🐞 = Fixed, 🔀 = Changed, 💥 = Breaking change.
 
-## [v2.0.1](https://github.com/ethereum/execution-spec-tests/releases/tag/v2.0.1) - 20XX-XX-XX: Unreleased
+## [v2.0.2](https://github.com/ethereum/execution-spec-tests/releases/tag/v2.0.2) - 202X-XX-XX: Untitled Release
 
 ### 🧪 Test Cases
 
 ### 🛠️ Framework
 
-- 🐞 State tests generated with transition forks no longer use the transition fork name in the fixture output, instead they use the actual enabled fork according to the state test's block number and timestamp ([#406](https://github.com/ethereum/execution-spec-tests/pull/406)).
-
 ### 📋 Misc
 
 ### 💥 Breaking Changes
```

Comment:
If we want to be super-clean, then the new template should be added after we tag v2.0.1, otherwise it'll show-up at the top of the tagged doc version here:
https://ethereum.github.io/execution-spec-tests/v2.0.0/CHANGELOG/